.travis.yml

```diff
@@ -119,19 +119,13 @@ script:
     else
       ../configure --enable-unit --enable-integration --with-crypto=$WITH_CRYPTO $CONFIGURE_OPTIONS CFLAGS=-I${PWD}/../osslinstall/usr/local/include LDFLAGS=-L${PWD}/../osslinstall/usr/local/lib
     fi
+  - make -j$(nproc) distcheck
   - |
     if [ "$CC" == "clang" ]; then
       scan-build --status-bugs make -j$(nproc) check
     else
       make -j$(nproc) check
     fi
-  - |
```

Comment:
Well, we still want to do this as a separate step in a new subdir, just remove the --disable-doxygen-doc from configure, please

Reply:
Will do!